sw/device/tests/csrng_kat_test.c

```diff
@@ -14,19 +14,19 @@
 
 OTTF_DEFINE_TEST_CONFIG();
 
-void test_ctr_drbg_ctr0(const dif_csrng_t *csrng) {
-  CHECK_DIF_OK(dif_csrng_uninstantiate(csrng));
-  csrng_testutils_fips_instantiate_kat(csrng, /*fail_expected=*/false);
-  csrng_testutils_fips_generate_kat(csrng);
+status_t test_ctr_drbg_ctr0(const dif_csrng_t *csrng) {
+  TRY(dif_csrng_uninstantiate(csrng));
+  TRY(csrng_testutils_fips_instantiate_kat(csrng, /*fail_expected=*/false));
+  TRY(csrng_testutils_fips_generate_kat(csrng));
+  dif_result_t res = kDifOk;
+  return INTO_STATUS(res);
 }
 
-bool test_main(void) {
+status_t test_main_status(void) {
```

Comment:
Again, no need for a `test_main_status`.

Reply:
if a test uses a testutil that now returns a `status_t`, just update how that `testutil` is called e.g., `CHECK_STATUS_OK(testutil(...))`

Reply:
Done